fe/fe-core/src/main/java/com/starrocks/catalog/MaterializedView.java

```diff
@@ -305,6 +307,17 @@ public void setLastRefreshTime(long lastRefreshTime) {
     // Maintenance plan for this MV
     private transient ExecPlan maintenancePlan;
 
+    // max delay duration to check which partitions to refresh, in this duration we use last
+    // `needToRefreshPartitionNamesCache` as the current `needToRefreshPartitionNames` result.
+    // NOTE: unit second.
+    private int maxMVRewriteStaleness = 0;
```

Comment:
what's the exactly behavior of this variable ? Decide the refresh behavior, or the rewrite staleness ?